documentation/FlowManager.md

```diff
@@ -22,16 +22,28 @@ different kinds of flows are:
 * **Update Preferences** (`CloudBasedFlowManager.js`) - used to update the preferences when the system is running in
   cloud-based mode. See [CloudBasedFlow](CloudBasedFlow.md) for more details
 
-## Reserved GPII Key "noUser"
+## Reserved GPII Keys
 
-The reserved special GPII key "noUser" is keyed into the system when there is not an actual key keyed in. This includes:
+### noUser
+
+The reserved GPII key "noUser" is automatically keyed into the system when there is not an actual key keyed in. This includes:
 
 * When GPII starts
 * Once an actual GPII key is keyed out
 
 The present of "noUser" key allows users to continue to change settings via QSS (Quick Strip Set) when no actual GPII
 key is keyed into the system.
 
+### reset
+
+The reserved GPII key "reset" is to be used with the flow manager login/logout APIs to reset the computer. These APIs are:
+
+GET /user/reset/proximityTriggered
```

Comment:
Because the "reset" token should immediately cause a logout of itself I think the only API which needs to be advertised is "login"